app/packs/src/apps/mydb/elements/details/wellplates/designerTab/WellOverlay.js

```diff
@@ -234,7 +234,7 @@ WellOverlay.propTypes = {
   removeSampleFromWell: PropTypes.func.isRequired,
   handleWellLabel: PropTypes.func.isRequired,
   handleColorPicker: PropTypes.func.isRequired,
-  selectedColor: PropTypes.func.isRequired,
+  //selectedColor: PropTypes.func.isRequired,
```

Comment:
Warum ist das hier auskommentiert worden?

Reply:
Gute Frage. Vielleicht durch das rebasen entstanden? Ich habe das wieder entfernt, da es in der aktuellen Version auch nicht auskommentiert ist.
Ich wollte da nur eine von den nervigen warnings in der console entfernen